packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.types.ts

```diff
@@ -126,6 +126,13 @@ export interface IHoverCardProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   target?: HTMLElement | string | null;
 
+  /**
+   * Optional target element to set event listener.
+   * This props is to separate the target to anchor hover card from the target to attach event listener.
+   * When no `eventListenerTarget` given, HoverCard will use `target`props or its root to set event listener.
+   */
+  eventListenerTarget?: HTMLElement;
```

Comment:
We could have the type of the new prop same as that of `target`. As in, include support for `string` so that developers can easily specify `eventListenerTarget` using query selector strings.